spec/requests/mutations/resend_confirmation_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'Resend confirmation' do
+  include_context 'with graphql query request'
+
+  let(:user)     { create(:user, confirmed_at: nil) }
+  let(:email)    { user.email }
+  let(:redirect) { Faker::Internet.url }
+  let(:query) do
+    <<-GRAPHQL
+      mutation {
+        userResendConfirmation(
+          email:"#{email}",
+          redirectUrl:"#{redirect}"
+        ) {
+          message
+          success
+        }
+      }
+    GRAPHQL
+  end
+
+  context 'when params are correct' do
+    it 'sends an email to the user with confirm url' do
+      expect { post_request }.to(
```

Comment:
we usually prefer this syntax
```ruby
expect { post_request }.to change(ActionMailer::Base.deliveries, :count).by(1)
```

Reply:
Got it, I'll fix this. I noticed VSCode complaining about Rubocop sometimes. Should I get that set up on my end? Is that something that would've warned me about that?

Reply:
not sure if rubocop rspec might prefer that syntax. But rubocop is a good thing to have

Reply:
Ok I'll look into getting it configured this weekend with my VSCode/Environment.